buildscript.txt

```diff
@@ -1,10 +1,10 @@
 filestore
---language=coffeescript
---node-version=6.9.5
---acceptance-creds=None
---dependencies=mongo,redis
+--acceptance-creds=
+--public-repo=True
+--language=es
+--node-version=10.17.0
+--dependencies=mongo,redis,s3
```

Comment:
Actually `filestore` does neither depend on mongo nor redis. It was incorrect before, but maybe change it now?

Reply:
Good point :+1: